aemeds/blocks/header/header.js

```diff
@@ -56,20 +55,22 @@ export default async function decorate(block) {
   try {
     block.append(
       section({
-        id: 'naas-header-old',
-        class: 'naas-header-old-section',
+        id: 'naas-header-v2',
+        class: 'cmp-nav__wrapper',
         'data-domain': dataDomain,
         'data-myaccount': 'hide',
         'data-search': 'hide',
         'data-sourceId': 'blogs',
         'data-lslinkshard': 'on',
+        'data-version': 'v2',
       }),
     );
 
     // load NaaS header code
     await Promise.all([
-      loadCSS(`${dataDomain}/nas/csi/header/v1/headerOldCSR.bundle.css`),
-      loadScript(`${dataDomain}/nas/csi/header/v1/headerOldCSR.bundle.js`),
+      loadCSS(`${dataDomain}/nas/csi/header/v2/headerV2CSR.bundle.css`),
+      loadCSS(`${dataDomain}/etc.clientlibs/ds/clientlibs/clientlib-arc-commons.min.css`),
```

Comment:
What I meant is, if it we shouldn't load additional CSS if it is not needed to render and based on what I've seen, the header looks ok even without this additional big CSS file.

Reply:
Yes, I have checked with Pranay from NaaS team who owning this. We should load this and this is required. Some styles are referenced from this file and it will still work without arc css. but we want all elements to follow arc styles. So in future we are changing some font color to different shade. we don't keep changing all references manually.

Earlier to understand its purpose, I have commented and checked. Then got confirmation from Pranay and this is required.

Reply:
My last agreement with Pranay and the NaaS team, prior to go live was that the arc css would not be loaded for NaaS, until there is a deeper integration with the ARC Design System across the whole project.

Reply:
I have confirmed with pranay. v2 is well integrated with arc than v1. We would need this import.